qa/suites/rgw/multisite/tasks/test_multi.yaml

```diff
@@ -16,3 +16,4 @@ tasks:
     config:
       reconfigure_delay: 60
     args: ['-a', '!bucket_sync_disable,!bucket_reshard'] # filter out disable/enable and reshard tests
+- interactive:
```

Comment:
we'll want to remove this bit before merge. i also didn't realize we were filtering out '!bucket_sync_disable,!bucket_reshard'. we should be running those!